src/Formio.js

```diff
@@ -650,7 +650,7 @@ export default class Formio {
       .then(() => Formio.pluginGet('staticRequest', requestArgs)
         .then((result) => {
           if (isNil(result)) {
-            return Formio.request(url, method, requestArgs.data, requestArgs.opts.header, requestArgs.opts);
+            return Formio.request(requestArgs.url, requestArgs.method, requestArgs.data, requestArgs.opts.header, requestArgs.opts);
```

Comment:
Did you mean to include this change? We can quickly merge the tree changes in but this is a major change to a foundational library and we may need to take a closer look at it.